### PR TITLE
Add kmwversion script to create static file

### PR DESCRIPTION
**Pull request by darcywong00**
Starts to address #89. Though `master` branch would be updating kmwversion.php, this would get merged to `staging` at end of sprint. I updated `kmwversion.sh` to follow the "flow" of kmwversion.php.

The deploy part of #89 would be handled on a separate PR. If we use a GHA, it should have Linux bashisms available...

### CODE REVIEW

File: metadata/kmwversions.json [hunk 1]
```diff
@@ -0,0 +1,2089 @@
+{
+    "versions": [
+        "17.0.132",
```
Comment (ermshiperete) [inline metadata/kmwversions.json:3]: Seems this list contains all version numbers twice...

Comment (darcywong00) [reply metadata/kmwversions.json:3]: good catch! I've tweaked to hopefully fix that

File: metadata/kmwversions.json
Comment (mcdurdin) [inline metadata/kmwversions.json:1]: This file should not be in the repo, as it should be built during deployment.

File: api/kmwversion.sh [hunk 2]
```diff
@@ -0,0 +1,45 @@
+#!//usr/bin/env bash
```
Comment (mcdurdin) [inline api/kmwversion.sh:1]:
```suggestion
#!/usr/bin/env bash
```

File: api/kmwversion.sh
Comment (mcdurdin) [inline api/kmwversion.sh:1]: These files should be in /deploy or something like that, not /api, which suggests an API surface for the site.

**Review by mcdurdin — CHANGES_REQUESTED** [review]

**Review by mcdurdin — APPROVED** [review]
This LGTM

**Comment by mcdurdin** [comment]
Might want a .gitignore for data files in metadata/

**Pull request merged**